src/freenet/pluginmanager/PluginManager.java

```diff
@@ -324,12 +308,14 @@ private static String pluginList(ArrayList<PluginInfoWrapper> wrappers) {
 	}
 
 	private String[] getConfigLoadString() {
-		ArrayList<String> v = new ArrayList<String>();
+		List<String> v = new ArrayList<String>();
+		for (PluginInfoWrapper pi : pluginWrappers) {
```

Comment:
Mostly because if I didn’t I would either have to let it stay in the `synchronized` block or would have to split up the `synchronized` block into two `synchronized` blocks… I’m not sure which solution is better. :)